csv_file_read_handler.py

```diff
@@ -0,0 +1,18 @@
+"""
```

Comment:
Choose a better name for the module.

Reply:
What's considered a good or better name?

Reply:
Could print_csv be a better name?

Reply:
I created the file to help me print the csv file cells cause I needed to print them in a specific way, so it only contains the printing function.

Reply:
so maybe include the word util in the name ?

Reply:
so woud csv_print_util be okay?

Reply:
great